### PR TITLE
[stable/locsut] Adapt locust helm chart to k8s compatibility.

### PR DESCRIPTION
**Pull request by stefanandres**
#### What this PR does / why we need it:

This PR changes the objects used in the locust helm charts and makes them k8s 1.16 compatible.

I've try to change as minimum as possible and still make the chart work. If you need/want any further changes, please tell.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by sekka1** [comment]
Will take a look.

**Comment by sekka1** [comment]
Sorry...still trying to find time to test this out.  Will update.

File: stable/locust/templates/worker-deploy.yaml [hunk 1]
```diff
@@ -69,3 +73,6 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
 {{- end }}
+{{- with .Values.worker.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
```
Comment (haugene) [inline stable/locust/templates/worker-deploy.yaml:78]: Getting a lint-error here: ```[ERROR] templates/: parse error at (locust/templates/worker-deploy.yaml:80): unexpected EOF```

You're missing an `{{- end }}` block at the end.

**Review by haugene — CHANGES_REQUESTED** [review]
Looks good to me overall. You need to add the {{- end }} block at the end of the worker-deploy template, but adding this it installs and works as expected.

**Comment by haugene** [comment]
Hello @stefanandres and sorry about the terrible delay on this PR. I've had it in the back of my mind and never getting to prioritise it. I've spun up a new cluster now and I'm going to try to review and get merged as many of the PRs here as I can this week.

Could you add the `{{- end }}` line as I've suggested and then I think all the tests will pass and we can mark this ready for merging 😄

**Comment by haugene** [comment]
The commit from this PR is now merged through #22186 with the lint error fix I mentioned.
This PR can be closed. Thank you @stefanandres for keeping the Chart up to date, sorry about the delay!

I can't close the PR, so either you can or the stale bot will do it at some point 😄

**Comment by stefanandres** [comment]
thanks!

**Pull request closed without merging**